styleguide/STYLEGUIDE.md

```diff
@@ -1,26 +1,24 @@
-## General
+## DuckDuckGo Android Style Guide
 * We care about clean code and aim to make this codebase as self-documenting and readable as possible.
-* We primarily use Kotlin and follow the standard coding conventions outlined in https://kotlinlang.org/docs/reference/coding-conventions.html
+* We've provided a coding style that can be imported into Android Studio.
```

Comment:
>We've provided a coding style that can be imported into Android Studio.

This duplicate line seems like a lingering artifact of the merge, since it's also included in the `Android Studio Setup` section?